tracks/csharp/exercises/nucleotide-count/mentoring.md

```diff
@@ -0,0 +1,57 @@
+### Reasonable solutions
+
+### Using a foreach loop
+
+```csharp
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public static class NucleotideCount
+{
+    public static IDictionary<char, int> Count(string sequence)
+    {
+        var result = new Dictionary<char, int> { {'A', 0}, {'C', 0}, {'G', 0}, {'T', 0} };
+        
+        foreach (var c in sequence)
+        {
+            if (result.ContainsKey(c)) result[c]++;
+            else throw new ArgumentException("Invalid Nucleotide Symbol");
+        }
+
+        return result;
+    }
+}
+```
+
+### Foreach with ToDictionary
+
+```csharp
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public static class NucleotideCount
+{
+    private static readonly char[] ValidNucleotideSymbols = {'A', 'C', 'G', 'T'};
+    
+    public static IDictionary<char, int> Count(string sequence)
+    {
+        var result = ValidNucleotideSymbols.ToDictionary(c => c, c => 0);
+        
+        foreach (var c in sequence)
+        {
+            if (ValidNucleotideSymbols.Contains(c)) result[c]++;
+            else throw new ArgumentException("Invalid Nucleotide Symbol");
+        }
+
+        return result;
+    }
+}
+```
+
+### Common Suggestions
+
+ * Iterating over `sequence` more than once wastes cycles, memory, etc. Often it isn't obvious that this is happening because methods like `ToList`, `GroupBy` or `OrderBy` will iterate a list without the student having to write a loop.
```

Comment:
Remove the list indicator here _or_ indent line [56](https://github.com/exercism/website-copy/pull/864/files#diff-6773c5801a8b5f93a7292c81cfadda3fR56) and [57](https://github.com/exercism/website-copy/pull/864/files#diff-6773c5801a8b5f93a7292c81cfadda3fR57).

The `Hence` and `Similarly` lines are suggestions based on this item.

Reply:
Lines 56 and 57 are further indented than 55 (two spaces), and display as sub-points in my markdown previewer.

Reply:
You're right. I don't know what happened there -- displays fine on GH too.